src/Contact.js

```diff
@@ -0,0 +1,36 @@
+import React, {PropTypes} from 'react';
+import RemoveContactButton from './RemoveContactButton.js';
+import ContactName from './ContactName.js';
+
+const Contact = props => {
+  return (
+    <li className="contact">
+      <div className="image-cropper">
+        <img src={props.avatar} alt="avatar" />
+      </div>
+      <div className="contact-info">
+        <ContactName
```

Comment:
Everything looks really good, but when I ran your project I noticed I could only select by clicking on the contact name. From a UX standpoint I think it could be better. First, whichever element you choose to attach the event listener to, I think you should use `cursor: pointer` to make it obvious to the user that they can click. Second, I recommend using the entire `<li>` element for the event listener. Then it's just a matter of adding `event.stopPropagation()' to any methods that are going to be passed to click listeners attached to children (like the remove button).